src/lib/comms/sol-netctl-impl-connman.c

```diff
@@ -0,0 +1,1159 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2015 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <systemd/sd-bus.h>
+#include <systemd/sd-event.h>
+
+#define SOL_LOG_DOMAIN &_sol_connman_log_domain
+#include <sol-log-internal.h>
+
+#include "sol-str-table.h"
+#include "sol-str-slice.h"
+#include "sol-util-internal.h"
+#include "sol-monitors.h"
+#include "sol-bus.h"
+#include "sol-netctl.h"
+
+SOL_LOG_INTERNAL_DECLARE_STATIC(_sol_connman_log_domain, "connman");
```

Comment:
rename this `"connman"` to netctl too. In output it will contain the filename as well, so it will be easy to know this comes from the -impl-connman.c.

The purpose of this string is to allow per-domain level, like: `export SOL_LOG_LEVELS=netctl:4` enables debug (4) for netctl domain.